cmd/swapcli/main.go

```diff
@@ -900,7 +900,11 @@ func runGetPastSwap(ctx *cli.Context) error {
 		fmt.Printf("Start time: %s\n", info.StartTime.Format(common.TimeFmtSecs))
 		fmt.Printf("End time: %s\n", endTime)
 		fmt.Printf("Provided: %s %s\n", info.ProvidedAmount.Text('f'), providedCoin)
-		fmt.Printf("Received: %s %s\n", info.ExpectedAmount.Text('f'), receivedCoin)
+		fmt.Printf("Received: %s %s", info.ExpectedAmount.Text('f'), receivedCoin)
```

Comment:
@stubbrn If you agree with the message that I wrote earlier that the actual received amount would be more helpful here, here's how you could calculate it:
```go
		receivedAmt := new(apd.Decimal)
		relayerFeeETH := coins.NewWeiAmount(info.RelayerFee).AsEther()
		_, err = coins.DecimalCtx().Sub(receivedAmt, info.ExpectedAmount, relayerFeeETH)
		if err != nil {
			return err
		}
```
Note: My other comment about storing the relayer fee in ETH instead of WEI would eliminate the 2nd line above.